### ISSUE TITLE
MISP custom auth configuration needs to be done by the users

### ISSUE THREAD

**Issue by iglocska** [melicertes/csp#50]
Considering that these settings are basically mandatory, why does a user have to configure it manually, leading to frustration and potential miscofigurations?

**Comment by thanosa75** [comment]
Is there a way to make this automated (eg. via cmdline?)

**Comment by iglocska** [comment]
Yes, I highly advise that you guys get familiar with the CLI / API commands supported by MISP. Go to event actions -> automation to get a full list where all of these are described. 

![image](https://user-images.githubusercontent.com/3668672/64772357-bf97b000-d550-11e9-8ddc-b4ddc5624986.png)


In this case what you are after is this CLI command:

The setting you are after in this case is this:

Set Setting: MISP/app/Console/cake Admin setSetting [setting] [value]
